spaces/aliases.json

```
@@ -22,4 +22,5 @@
   "macaronswap": "macaronswap.eth",
   "pepemon": "pepedontdump.eth",
   "akropolis-delphi": "akropolis.eth"
+  "rally": "rallygov.eth"
```

A comma is missing at the end of the line above your sub-domain.
```suggestion
  "akropolis-delphi": "akropolis.eth",
  "rally": "rallygov.eth"
```